filters/shedder/admission_test.go

```diff
@@ -117,10 +122,37 @@ func TestAdmissionControl(t *testing.T) {
 		N:                          20,
 		pBackendErr:                0.8,
 		pExpectedAdmissionShedding: 0.91,
+	}, {
+		msg:                        "inactive mode with large error rate and bigger than threshold will pass all traffic",
+		mode:                       "inactive",
+		d:                          10 * time.Millisecond,
+		windowsize:                 5,
+		minRequests:                10,
+		successThreshold:           0.9,
+		maxrejectprobability:       0.95,
+		exponent:                   1.0,
+		N:                          20,
+		pBackendErr:                0.8,
+		pExpectedAdmissionShedding: 0.0,
```

Comment:
I just trust these numbers make sense :)

Reply:
I basically copied the last test that has `pExpectedAdmissionShedding: 0.91,` and for "inactive" I expect  `pExpectedAdmissionShedding: 0`